tests/src/psa_crypto_helpers.c

```diff
@@ -74,21 +74,9 @@ const char *mbedtls_test_helper_is_psa_leaking(void)
 
     mbedtls_psa_get_stats(&stats);
 
-#if !defined(MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG) &&                        \
-    defined(MBEDTLS_CTR_DRBG_C) &&                                      \
-    defined(MBEDTLS_CTR_DRBG_USE_PSA_CRYPTO)
-    /* When AES_C is not defined and PSA does not have an external RNG,
-     * then CTR_DRBG uses PSA to perform AES-ECB. In this scenario 1 key
-     * slot is used internally from PSA to hold the AES key and it should
-     * not be taken into account when evaluating remaining open slots. */
     if (stats.volatile_slots > 1) {
```

Comment:
Unless I'm misunderstanding, I think this should be:
```suggestion
    if (stats.volatile_slots > MBEDTLS_TEST_PSA_INTERNAL_KEYS) {
```